src/Runner.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -522,15 +524,20 @@ private async Task<bool> IsHostedServer(string serverUrl, VssCredentials credent
 
         private async Task<GitHubAuthResult> GetTenantCredential(string githubUrl, string githubToken)
         {
-            var gitHubUrl = new UriBuilder(githubUrl);
-            var githubApiUrl = $"https://api.{gitHubUrl.Host}/repos/{gitHubUrl.Path.Trim('/')}/actions-runners/registration";
+            var githubApiUrl = "https://api.github.com/actions/register-runner";
```

Comment:
you may want to persist `api.{gitHubUrl.Host}` as before, since in local dotcom dev env it's github.localhost